modules/telestion-api/src/main/java/de/wuespace/telestion/api/message/header/Information.java

```diff
@@ -0,0 +1,164 @@
+package de.wuespace.telestion.api.message.header;
+
+import de.wuespace.telestion.api.message.header.serialization.NoSerializationException;
+import de.wuespace.telestion.api.message.header.serialization.SerializationUtils;
+import de.wuespace.telestion.api.utils.NoSuchPrimitiveTypeException;
+import io.vertx.core.MultiMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.RecordComponent;
+import java.util.*;
+
+/**
+ * The base class for all information records for header metadata information.
+ *
+ * @author Cedric Boes (@cb0s), Ludwig Richter (@fussel178), Pablo Klaschka (@pklaschka)
+ */
+public interface Information {
```

Comment:
Information on what? Just call it Header, HeaderInfo, Metadata? I don't know but information is a weak name.